src/NuGet.Core/NuGet.Commands/Strings.resx

```diff
@@ -1078,4 +1078,10 @@ Non-HTTPS access will be removed in a future version. Consider migrating to 'HTT
     <value>Error occurred while getting package vulnerability data: {0}</value>
     <comment>{0} is an error message</comment>
   </data>
+  <data name="Error_InvalidNuGetAuditModeValue" xml:space="preserve">
+    <value>Invalid NuGetAuditMode value '{0}'. Expected values: {1}</value>
+    <comment>Don't translate 'NuGetAuditLevel'
```

Comment:
```suggestion
    <comment>Don't translate 'NuGetAuditMode'
```

Reply:
This was marked resolved but it is still showing NuGetAuditLevel .... ?